dockerfile_parse/parser.py

```diff
@@ -290,7 +297,7 @@ def _instruction_getter(self, name, env_replace):
         if name != 'LABEL' and name != 'ENV':
             raise ValueError("Unsupported instruction '%s'", name)
         instructions = {}
-        envs = {}
+        envs = self.parent_env
```

Comment:
Do we need to clone `self.parent_env` ? After this method is called it'll contain additional items.